packages/cli/src/commands/deploy/helper.ts

```diff
@@ -186,6 +211,18 @@ export const promptNewName = async (allFieldPlugins: FieldType[]) => {
   return name
 }
 
+export const confirmOptionsUpdate = async () => {
+  const { confirmed } = await betterPrompts<{
+    confirmed: boolean
+  }>({
+    type: 'confirm',
+    name: 'confirmed',
+    message: `The plugin's options is going to also be updated based on the ones found in your manifest file. Do you want to proceed?`,
```

Comment:
What do you think about this message?

I don't know if there would be a better way to rephrase it.

Reply:
```suggestion
    message: `The plugin options are going to be replaced by the options specified inside the manifest file. Do you want to proceed?`,
```

Maybe?

Reply:
Thanks for the suggestion @BibiSebi. 🙌 

I found yours better.